### DISCUSSION TITLE
Game Requests

### DISCUSSION THREAD

**Discussion by AnimMouse** [AnimMouse/psn-pkg-torrents#8] [Ideas]
Request games here.

**Comment by AGASTEOFICIAL** [comment]
L.A Noire For Folder

**Reply by AnimMouse** [reply]
Done. [L.A. Noire PS3 NPUB30679.torrent](https://github.com/AnimMouse/psn-pkg-torrents/blob/main/PS3/L.A.%20Noire%20PS3%20NPUB30679.torrent)

**Comment by farhadabdul56** [comment]
Please can you make carx drift ps4pkg

**Reply by AnimMouse** [reply]
As of this comment, you can't run any PS4 PKG downloaded from Sony servers as we haven't defeated the PS4's DRM yet and the only way to play is to use fake PKGs.

**Comment by Xtremely-Doped-Punk** [comment]
WWE 2k23 ps4 pkg please

**Reply by AnimMouse** [reply]
As of this comment, you can't run any PS4 PKG downloaded from Sony servers as we haven't defeated the PS4's DRM yet and the only way to play is to use fake PKGs.

**Comment by Enocomy** [comment]
gow 3 ps3 plz


> Request games here.

**Reply by AnimMouse** [reply]
Unfortunately, God of War III has not been released in PlayStation Store, it is only released in Blu-ray form. So there is no PSN PKG for God of War III.

**Reply by Enocomy** [reply]
It's fine yk you are one of them real guys responding every reply for 4 years i really respect you

**Comment by alan72867801-collab** [comment]
God of war 2018 PS4 please.
